pkgs/top-level/all-packages.nix

```diff
@@ -24220,6 +24220,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa CoreGraphics ForceFeedback OpenAL OpenGL;
   };
 
+  blender_2_9 = callPackage  ../applications/misc/blender/2.9.nix {
```

Comment:
```suggestion
  blender_2_93 = callPackage  ../applications/misc/blender/2.93.nix {
```

or

```suggestion
  blender_2 = callPackage  ../applications/misc/blender/2.nix {
```

Reply:
> Is there a reason to keep blender 2.x? If 3.0 builds on darwin, we can just remove 2.x

Blender 3.0 removed support for OpenCL rendering, which i bet many of the amd gpu users would miss.